src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const CALC_COUNT = PRICE_PER_DAY * days;
+  const SAIL_FOR_THREE_MORE = 20;
+  const SAIL_FOR_SEVEN_MORE  = 50;
```

Comment:
The constants `SAIL_FOR_THREE_MORE` and `SAIL_FOR_SEVEN_MORE` could be named more descriptively to reflect their purpose in the context of this function. Consider renaming them to `THREE_DAY_DISCOUNT` and `SEVEN_DAY_DISCOUNT` respectively, to make it clear that these are discounts applied based on the number of rental days.